### DISCUSSION TITLE
i have a question to ask you.

### DISCUSSION THREAD

**Discussion by Outfit8TSB** [Dans-Plugins/Medieval-Factions#1807] [Q&A] [answered]
can factions place blocks on other factions or unclaimed zone(Wilderness)?

**Comment by dmccoystephenson** [comment, marked as answer]
Hey @Outfit8TSB,

Yes, there are scenarios where factions can allow players in other factions to access their territory. As for unclaimed land, by default it can be interacted with by anyone.

**Reply by Outfit8TSB** [reply]
that's nice, i am a guy who wants to run a anarchy server with that plugin, anyway thanks.

**Reply by dmccoystephenson** [reply]
Absolutely, please let me know if you have any more questions.
